subsys/bluetooth/host/audio/chan.c

```diff
@@ -1384,7 +1384,8 @@ static int bt_audio_set_base(const struct bt_audio_broadcast_source *source,
 
 int bt_audio_broadcast_source_create(struct bt_audio_chan *chan,
```

Comment:
It will get slightly shorter when we remove the channel lists, but I guess we can easily move the input parameter validation into a separate function.

Reply:
Update: The input parameter validation actually doesn't contain much at the moment, and didn't really make much sense to move.

Reply:
It is a quite straight forward function, so it is not really a problem.